lib/js/components/ProgressBar/ProgressBar.vue

```diff
@@ -12,7 +12,7 @@
 					'-ds-medium': labelTextSize === PROGRESS_BAR_LABEL_TEXT_SIZES.MEDIUM,
 					'-ds-ellipsis': labelTextEllipsis,
 				}"
-				:title="labelTextEllipsis ? labelText : null"
+				:title="labelTextEllipsis ? labelText : undefined"
```

Comment:
This is not an component, so I wonder why even this title is there, and why it is dependant on `labelTextEllipsis` text is provided anyway below